src/components/ex-live-search-results/ex-live-search-results.scss

```diff
@@ -0,0 +1,101 @@
+.ex-live-search-results {
+  @apply overscroll-contain absolute hidden overflow-y-auto bg-white;
+
+  box-shadow: 0px 3px 45px #0000002c;
+  max-height: calc(100vh - 200px);
```

Comment:
I believe there is a maximum of 5 search results. I would not limit the height of the box.

![image](https://user-images.githubusercontent.com/88790097/141469461-8aa63a47-f874-4eb2-ab08-0ba72aac7e79.png)

Reply:
removed max height